### PR TITLE
Prevent display of Workflow and Aging Pages Reports without appropriate permissions

### LINKED ISSUE #8845: Workflow reports showing for user with limited permissions.
### Issue Summary

I have a user belonging to a group with very limited permissions and no permissions assigned at the user level. The Wagtail admin is correctly limiting which snippet models this user can see, but I am surprised they can see Workflow and Aging Pages reports. I expected the admin menu to only show "Snippets". 

Screen Shot 2022-07-12 at 12 25 18 PM

Screen Shot 2022-07-12 at 12 24 44 PM

Screen Shot 2022-07-12 at 12 25 04 PM

### Technical details

Python version: 3.8.5
Django version: 4.0.6
Wagtail version: GitHub main as of July 11 SHA: https://github.com/wagtail/wagtail/commit/22cf8c240d5459b2837a67319fb385f11f54492d

### PR DESCRIPTION
**Pull request by rohitsrma**
Fixes #8845 In this PR the workflow and aging pages report visibility has been restricted for users without appropriate permissions.

_Please check the following:_

-   [x] Do the tests still pass?[^1]
-   [x] Does the code comply with the style guide?
    -   [x] Run `make lint` from the Wagtail root.
-   [x] For Python changes: Have you added tests to cover the new/fixed behaviour?
-   [ ] For front-end changes: Did you test on all of Wagtail’s supported environments?[^2]
    -   [ ] **Please list the exact browser and operating system versions you tested**:
    -   [ ] **Please list which assistive technologies [^3] you tested**:
-   [ ] For new features: Has the documentation been updated accordingly?

**Please describe additional details for testing this change**.

[^1]: [Development Testing](https://docs.wagtail.org/en/latest/contributing/developing.html#testing)
[^2]: [Browser and device support](https://docs.wagtail.org/en/latest/contributing/developing.html#browser-and-device-support)
[^3]: [Accessibility Target](https://docs.wagtail.org/en/latest/contributing/developing.html#accessibility-targets)

### CODE REVIEW

File: wagtail/admin/wagtail_hooks.py [hunk 1]
```diff
@@ -863,7 +867,11 @@ def is_shown(self, request):
 
 class AgingPagesReportMenuItem(MenuItem):
     def is_shown(self, request):
-        return getattr(settings, "WAGTAIL_AGING_PAGES_ENABLED", True)
+        return getattr(
+            settings, "WAGTAIL_WORKFLOW_ENABLED", True
```
Comment (gasman) [inline wagtail/admin/wagtail_hooks.py:871]: The aging pages report isn't related to workflows, so I think this part of the check can be removed, just leaving the PagePermissionPolicy check.

Comment (rohitsrma) [reply wagtail/admin/wagtail_hooks.py]: Apologies I missed that, corrected now!!

**Review by gasman — APPROVED** [review]
All looks good to me - thanks @rohitsrma!

**Comment by gasman** [comment]
Merged in 13048eab793d80f6bc7a32aa14a1c3511f494fa9

**Pull request closed without merging**